main/src/ca/uwaterloo/flix/api/lsp/provider/completion/KeywordCompleter.scala

```diff
@@ -29,25 +29,25 @@ object KeywordCompleter {
     */
   def getDeclKeywords: Iterable[Completion] =
     List(
-      Completion.KeywordCompletion("def"      , Priority.highest("def")),
-      Completion.KeywordCompletion("pub"      , Priority.higher("pub")),
-      Completion.KeywordCompletion("enum"     , Priority.high("enum")),
-      Completion.KeywordCompletion("type"     , Priority.high("type")),
-      Completion.KeywordCompletion("instance" , Priority.high("instance")),
-      Completion.KeywordCompletion("mod"      , Priority.low("mod")),
-      Completion.KeywordCompletion("eff"      , Priority.lower("eff")),
-      Completion.KeywordCompletion("struct"   , Priority.lower("struct")),
-      Completion.KeywordCompletion("sealed"   , Priority.lowest("sealed")),
-      Completion.KeywordCompletion("trait"    , Priority.lowest("trait")),
-      Completion.KeywordCompletion("import"   , Priority.lowest("import")),
+      Completion.KeywordCompletion("def"      , Priority.Highest),
+      Completion.KeywordCompletion("pub"      , Priority.Higher),
+      Completion.KeywordCompletion("enum"     , Priority.High),
+      Completion.KeywordCompletion("type"     , Priority.High),
+      Completion.KeywordCompletion("instance" , Priority.High),
+      Completion.KeywordCompletion("mod"      , Priority.Low),
+      Completion.KeywordCompletion("eff"      , Priority.Lower),
+      Completion.KeywordCompletion("struct"   , Priority.Lower),
+      Completion.KeywordCompletion("sealed"   , Priority.Lowest),
+      Completion.KeywordCompletion("trait"    , Priority.Lowest),
+      Completion.KeywordCompletion("import"   , Priority.Lowest),
     )
```

Comment:
Much better. Now we are getting somewhere. 

Are the priorities here based on how frequently the keywords are?

Reply:
OH, sorry. Yes, the priorities here are. I thought you had referenced the priorities of builtin types

Reply:
but it's based on the priorities compared to all the other keywords in the list, not just the ones that they share a prefix with. That's the issue you pointed out in #8419. Currently, I'm working on that